dace/sdfg.py

```diff
@@ -2317,6 +2317,51 @@ def traverse(node):
         # Return node that corresponds to current edge
         return traverse(tree_root)
 
+def trace_nested_access(node, state, sdfg):
+    """ Given an AccessNode in a nested SDFG, trace the accessed memory
+        back to the outermost scope in which it is defined.
+
+        :param node: An access node.
+        :param state: State in which the access node is located.
+        :param sdfg: SDFG in which the access node is located.
+        :return: A list of scopes (node, state, sdfg) in which the given
+                 data is accessed, from outermost scope to innermost scope.
+        """
+    if node.access == dace.dtypes.AccessType.ReadWrite:
```

Comment:
Do you really want to rely on access types like this? there might be cases that an output node is accessed as readwrite in the middle of the nested SDFG (first written, then read, then written to again).

Reply:
I don't need this and I don't have a test case for it, so I won't spend time on it now :-) If we ever need it we can add this feature